docs/openapi/components/schemas/credentials/EInvoiceCredential.yml

```diff
@@ -0,0 +1,1048 @@
+$linkedData:
+  term: EInvoiceCredential
+  '@id': https://w3id.org/traceability#EInvoiceCredential
+title: e-Invoice Credential
+tags:
+  - Steel
+description: >-
+  Narrowly defined specific invoice for steel
```

Comment:
Include the link to the spec: 
MX CFDI v4.0 https://learn.microsoft.com/en-us/dynamics365/business-central/localfunctionality/mexico/electronic-invoicing